.github/workflows/maven.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-18.04]
+        os: [ubuntu-18.04, windows-2022, macos-11]
```

Comment:
Actually ubuntu-18.04 is deprecated in Github Action(For more information, see [here](https://github.blog/changelog/2022-08-09-github-actions-the-ubuntu-18-04-actions-runner-image-is-being-deprecated-and-will-be-removed-by-12-1-22/)), reserve it for the current status checker.